ADR/0018-apps-continuous-perf-testing.md

```diff
@@ -0,0 +1,72 @@
+# 18. Continuous Performance Testing (CPT) of Apps in Stonesoup
+
+Date: 2023-03-10
+
+## Status
+
+In consideration
+
+## Context
+
+In general, performance testing is just another form of testing that helps application teams to ensure there are no regressions in their code and that their application behaves as expected.
+
+The IntegrationTestScenario pipelines in StoneSoup are not suitable for full-blown performance and scale testing (that usually takes lots of time and involves human analysis), but are a good place for a quick and small-scale regression test.
```

Comment:
Why are they not sufficient? Can we expound upon this a little bit to inform what classes of problems might need to investigate a different testing path?

Reply:
Added some more info. Hope it is better now.